third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -344,37 +426,65 @@ function buildProtocolResult({ result, exceptionDetails }) {
   return { result: protocolResult };
 }
 
-function Pause_evaluateInFrame({ frameId, expression }) {
-  const frames = getStackFrames();
-  const index = +frameId;
-  assert(index < frames.length);
-  const frame = frames[index];
 
-  const rv = doEvaluation();
-  return buildProtocolResult(rv);
+function Pause_evaluateInFrame({ frameId, expression }) {
+  try {
+    const result = window.DevOnly?.tryEvalDev(expression, frameId);
```

Comment:
Should this and the other changes to Pause_evaluateInFrame be reverted as well?

Reply:
Ok, I'll keep the `DevOnly.tryEvalDev` stuff in private files for now (since I use it sometimes).
I'll consider advocating for it, once/if I have a sufficiently strong case in the future.

Removed [here](https://github.com/replayio/chromium/commit/51766371372d228ea0ed3ea5c68c949891ee8960).